docs/src/api/params.md

```diff
@@ -218,6 +218,15 @@ Toggles bypassing page's Content-Security-Policy.
 
 Emulates consistent viewport for each page. Defaults to an 1280x720 viewport. `null` disables the default viewport.
 
+## context-option-viewport-csharp
```

Comment:
nit: rename to `csharp-context-option-viewport` to match other keys where the language goes first, e.g. `python-context-option-viewport`. It'd arguably be better to make language a suffix so that all overload definitions are grouped together but let's be consistent with the current approach, we can rename all of them at once if need be.